src/main/scala/temple/builder/DatabaseBuilder.scala

```diff
@@ -19,21 +19,22 @@ object DatabaseBuilder {
     }.toSeq
 
     val (colType, constraints) = attribute.attributeType match {
```

Comment:
Can we rename `constraints` to something more specific, to justify it not including `valueConstraints`?